src/__tests__/testUtils.ts

```diff
@@ -41,3 +41,31 @@ test('createErrorResponse', () => {
     },
   });
 });
+
+export function waitSeconds(seconds: number) {
+  return new Promise(resolve => {
+    setTimeout(resolve, seconds * 1000);
+  });
+}
+
+export async function retryInSeconds<O>(
+  func: () => Promise<O>,
+  secondsBetweenRetries = 3,
+  statusCodeToRetry = 404,
+  finishAfterSeconds: number = 300
+): Promise<O> {
+  const timeStart = Date.now();
+  while (Date.now() - timeStart < finishAfterSeconds * 1000) {
+    try {
+      return await func();
+    } catch (error) {
+      if (Number(error.status) !== statusCodeToRetry) {
```

Comment:
`error.status` is always a number?

Reply:
typescript doesn't know what it is in here
but I can make a type and catch only it, sounds ok?

Reply:
Ok 👍